CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `mq_timedreceive` to `::nix::mqueue`.
   ([#1966])(https://github.com/nix-rust/nix/pull/1966)
 - Added `LocalPeerPid` to `nix::sys::socket::sockopt` for macOS. ([#1967](https://github.com/nix-rust/nix/pull/1967))
+- Added `getsyscallinfo` to `nix::sys::ptrace` for Linux.
+  ([#2006](https://github.com/nix-rust/nix/pull/2006))
```

Comment:
We have changed our changelog mode, please see [`CONTRIBUTING.md`](https://github.com/nix-rust/nix/blob/master/CONTRIBUTING.md#pull-requests) on how to add a changelog.